<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Afinní šifra</string>
  </property>
  <property name="locale">
   <locale language="Czech" country="CzechRepublic"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutTop">
       <item>
        <widget class="QGroupBox" name="groupBoxInput">
         <property name="title">
          <string>Vstup</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>29</y>
            <width>301</width>
            <height>191</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayoutInput">
           <item row="0" column="0">
            <widget class="QLabel" name="labelInput">
             <property name="text">
              <string>Zadejte text k zašifrování:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="textEditInput"/>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayoutDoStuff">
             <item>
              <widget class="QPushButton" name="buttonDoStuff">
               <property name="text">
                <string>Proveď</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="spacerDoStuff">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxSettings">
         <property name="title">
          <string>Nastavení</string>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>305</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayoutOption">
             <item row="3" column="0">
              <widget class="QRadioButton" name="radioButtonDecrypt">
               <property name="text">
                <string>Rozšifrovat</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="radioButtonEncrypt">
               <property name="text">
                <string>Zašifrovat</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelOption">
               <property name="text">
                <string>Vyberte možnost:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayoutKeys">
             <item row="1" column="0">
              <widget class="QLabel" name="labelKeyA">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>A</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBoxKeyB">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>9999999</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelKeyB">
               <property name="text">
                <string>B</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelKeys">
               <property name="text">
                <string>Zvolte klíče:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBoxKeyA">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>9999999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxOutput">
       <property name="title">
        <string>Výstup</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>29</y>
          <width>671</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayoutOutput">
         <item row="0" column="0">
          <widget class="QLabel" name="labelOutput">
           <property name="text">
            <string>Zašifrovaný text:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPlainTextEdit" name="textEditOutput"/>
         </item>
         <item row="1" column="1">
          <widget class="QPlainTextEdit" name="textEditFiltered">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelFiltered">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Vyfiltrovaný text:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
